react-app/src/components/common/Buttons/IconButton.tsx

```diff
@@ -9,11 +10,21 @@ interface IconButtonProps
   > {
   icon: IconType;
   size: number;
+  title?: string;
+  tooltip?: React.ReactNode;
```

Comment:
I think we should give this a better name to indicate these props are for the tooltip component

Reply:
Maybe tooltipText or tooltipContent?

Reply:
think so yea